tools/rln_keystore_generator/rln_keystore_generator.nim

```diff
@@ -30,6 +31,12 @@ when isMainModule:
 
   trace "configuration", conf = $conf
 
+  let baseDir = conf.rlnRelayCredPath.splitFile().dir
+  if baseDir.len > 0 and not existsDir(baseDir):
+    let errMsg = "The next folder doesn't exist: " & baseDir
+    error "failure missing credentials folder", error=errMsg
+    quit(1)
+
```

Comment:
When running for the first time is the dir created before this check? Otherwise it can never pass.

Reply:
Yes, the folder should be created manually beforehand.

Reply:
while I agree that the check should be made, we should not create the directory, it is not the concern of the application to do it